src/azure-cli/azure/cli/command_modules/network/_params.py

```diff
@@ -642,6 +642,7 @@ def load_arguments(self, _):
         c.argument('floating_ip', help='Enable floating IP.', arg_type=get_three_state_flag())
         c.argument('idle_timeout', help='Idle timeout in minutes.', type=int)
         c.argument('protocol', help='Network transport protocol.', arg_type=get_enum_type(TransportProtocol))
+        c.argument('private_ip_address_version', min_api='2019-04-01', help='The private IP address version to use.', default=IPVersion.ipv4.value if IPVersion else '')
```

Comment:
According the issue, you should set default value as IPv6, right? But here your default value is ipv4?

Reply:
I don't think I should set the default value to IPv6. IPv6 is under public preview. Where do you find we should set this as IPv6?

Reply:
https://github.com/Azure/azure-cli/issues/10018#issuecomment-520538279

Reply:
well. I treat it as the answer to my question `Which kind of settings I should do in order to support IPv6?`

Reply:
So what about provide enum value for this parameter?like [Ipv4, Ipv6] and declare the default value is IPv4.

Reply:
This is the help info `The private IP address version to use.  Allowed values: IPv4, IPv6.  Default: IPv4.`

Reply:
Looks like it will automatically treat is as an enum type.